network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_feature_block.html

```diff
@@ -0,0 +1,27 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+<div class="container">
+<div class="row">
+
+  <div class="d-xl-block feature mb-md-5">
```

Comment:
Can you add some context as to why are using Bootstrap utilities instead of Tailwind?